app/javascript/stylesheets/_variables_dsfr.scss

```diff
@@ -0,0 +1,11 @@
+// override the default bootstrap grid breakpoints to match the DSFR ones
+// this is temporary while we still use responsive classes from bootstrap
+// cf https://www.systeme-de-design.gouv.fr/fondamentaux/grille-et-points-de-rupture
+
+$grid-breakpoints: (
+  xs: 0,
+  sm: 576px,
+  md: 768px,
+  lg: 992px,
+  xl: 1248px // this is actually the only one that is different
+);
```

Comment:
est-ce que ce bout de code aurait sa place dans `app/javascript/stylesheets/structure/_bootstrap_overrides.scss` ? Ça nous permettrait de garder un seul fichier à supprimer quand on abandonne bootstrap, et c'est effectivement des overrides de bootstrap

Reply:
bien vu, j’ai fait plusieurs itérations et en effet maintenant ce fichier n’a plus trop de sens. je déplace dans bootstrap_overrides. merci !